docs/source/using/profiles.adoc

```diff
@@ -63,4 +82,41 @@ To delete a profile, run:
 $ minishift profile delete PROFILE_NAME
 ----
 
-NOTE: The default profile 'minishift'  can not be deleted.
+NOTE: The default profile *minishift* cannot be deleted.
+
+[[example-workflows-profiles]]
+== Example Workflows
+
+To create a new profile *foo* with specific configurations, run:
+
+----
+$ minishift profile set foo
+$ minishift config set memory 4GB
+$ minishift config set cpus 2
+$ minishift addon enable anyuid
+$ minishift start
+----
+
+Alternate method to create the profile *foo*:
+
+----
+$ minishift --profile foo config set memory 4GB
+$ minishift --profile foo config set cpus 2
+$ minishift --profile foo addon enable anyuid
+$ minishift --profile foo minishift start
+----
+
+Note: Once you have started multiple profiles, you should use the xref:switching-profiles[`profile set`] command to switch between them.
+
+[[global-config-profiles]]
+== Global Configuartion For profiles
+
+Currently, we cannot set any global configuration for profiles.
+If you want to apply a common configuration to all your profiles then you need to apply the settings for each profile separately.
+As a workaround, xref:../using/basic-usage.adoc#environment-variables[environment variables] can be used.
```

Comment:
"In some cases  xref:../using/basic-usage.adoc#environment-variables[environment variables] can be used."

It won't work for all settings. Needs to be elaborated a bit more.